DESCRIPTION

```diff
@@ -18,7 +18,7 @@ Imports:
     storr
 Suggests:
     knitr,
-    orderly (>= 0.5.11),
+    orderly (>= 0.4.7),
```

Comment:
this looks like a merge failure (though it should be .14 I think)

Reply:
Updated to 0.5.14 in latest commit.

Reply:
0.5.14 is the version of the Orderly Master branch at the moment, but the error in Travis is:

checking package dependencies ... ERROR
Package required and available but unsuitable version: ‘orderly’

Reply:
Have updated .travis.yml to point to orderly master (which is 0.5.14)